tools/jit/gen_jit_dispatch.py

```diff
@@ -85,127 +84,123 @@ def is_jit_op(decl):
 
 
 def gen_jit_dispatch(declarations, out):
-    # We need to add methods implemented manually in TensorImpl
-    tensor_impl_methods = [{
-        'name': name,
-        'api_name': name,
-        'method_of': ['Tensor'],
-        'arguments': [{'name': 'self', 'simple_type': 'Tensor'}],
-    } for name in ['sizes', 'strides', 'dim']]
-    aten_decls = load_aten_declarations(declarations) + tensor_impl_methods
-    jit_decls = [d for d in aten_decls if is_jit_op(d)]
+    ops = {}
 
     def is_tensor_arg(arg):
         return arg['simple_type'] in {'Tensor', 'TensorList'}
 
-    ops = {}
-    names = set()
-    for decl in jit_decls:
+    def get_invocation(decl, args):
+        if 'namespace' in decl['method_of']:
+            return CALL_NAMESPACE.substitute(name=decl['name'], args=args)
+        else:
+            return CALL_METHOD.substitute(name=decl['name'], first=args[0], args=args[1:])
+
+    def emit_decl_variant(decl, is_real_input, has_tensorlist):
+        kw_assignments = []
+        attr_names = []
+        pos_assignments = []
+        arguments = []
+
+        if has_tensorlist:
+            kw_assignments.append('size_t varargs_length = node->inputs().size();')
+            # arguments look like: [tensor list], other, static, inputs
```

Comment:
it took me a moment to figure out that `other, static, inputs` doesn't mean that these are 3 sections of inputs 😛 how about `[tensor_list], arg1, arg2, ...`?